docs/getting-started/example-deployment/project-setup.md

```diff
@@ -0,0 +1,231 @@
+# Example Deployment
+
+This guide will run through the start up and deployment of a basic Gaffer instance. It will cover
+how to write a basic Gaffer Schema from scratch along with using the pre-made containers to run the
+Gaffer rest API and Accumulo based data store.
+
+!!! warning
+    Please be aware that the example is only intended to demonstrate the core Gaffer concepts it is
+    not a production example. Various additional configuration to Accumulo and the HDFS set up would
+    be required to make this production ready and likely be specific to your infrastructure.
+
+## The Example Graph
+
+For this basic example we will attempt to recreate the graph in the following diagram consisting of
+two nodes (vertexes) with one directed edge between them.
+
+```mermaid
+graph LR
+    A(["Person
+
+        name: marko
+        age: 29"])
+    --
+    "Created
+     weight: 0.4"
+    -->
+    B(["Software
+
+        name: lop
+        lang: java"])
+```
+
+This data describes one individual and a single piece of software that has been created by that
+individual. The data will be loaded into the graph from a CSV file that follows the Neo4j export
+syntax, this demonstrates how Gaffer can be used and how it can interact, model and query data from
+other popular graph databases. Even with this basic graph we should be able to start building
+queries to ask questions like "Who created the software called 'lop'?" and "How much did 'marko'
+contribute to the software called 'lop'?" etc.
+
+To go with the diagram above the following CSV file (both raw and rendered are provided) represents
+the graph in [Neo4j syntax](https://neo4j.com/labs/apoc/4.4/export/csv/#export-database-csv).
+
+!!! note ""
+    Please note that Gaffer often requires additional information about the data, such as
+    `:String` on the column headers, to help with typing of the values. This is demonstrated below
+    in the raw file.
```

Comment:
For this, it might be useful to link to the "OpenCypher Formats" section of `csv.md` as that's what these extra bits are.